controllers/cloudstackmachine_controller.go

```diff
@@ -54,6 +54,7 @@ var (
 // +kubebuilder:rbac:groups=cluster.x-k8s.io,resources=machines;machines/status,verbs=get;list;watch
 // +kubebuilder:rbac:groups=cluster.x-k8s.io,resources=machinesets,verbs=get;list;watch
 // +kubebuilder:rbac:groups=controlplane.cluster.x-k8s.io,resources=kubeadmcontrolplanes,verbs=get;list;watch
+// +kubebuilder:rbac:groups="",resources=events,verbs=create;patch
```

Comment:
should we restrict this to only the capc group?

Reply:
The group of event is "" (empty string), means it is inside core like `pod`
If we change it to `// +kubebuilder:rbac:groups="**infrastructure.cluster.x-k8s.io**",resources=events,verbs=create;patch`, resource events is not a valid value inside that groups.